snmp/tests/compose/data/generic_host.snmprec

```diff
@@ -1,4 +1,4 @@
-1.3.6.1.2.1.25.1.1|67|803137913
+1.3.6.1.2.1.25.1.1.0|67|803137913
```

Comment:
This is likely a typo, since scalar OID should end with `.0`. 

cc @mgarabed (I think you initialled authored this file)